### PR TITLE
Added flag for choosing authors source & no fallback for TME by default

### PR DESCRIPTION
**Pull request by tosan88**

### CODE REVIEW

File: service/suggester.go [hunk 1]
```diff
@@ -76,49 +82,55 @@ func NewAuthorsSuggester(authorsSuggestionApiBaseURL, authorsSuggestionEndpoint
...
+	falconResp, err := suggester.FalconSuggester.GetSuggestions(payload, tid)
 	if err != nil {
 		if err == NoContentError {
 			log.WithTransactionID(tid).WithField("tid", tid).Warn(err.Error())
 		} else {
 			log.WithTransactionID(tid).WithField("tid", tid).WithError(err).Error("Error calling Falcon Suggestions API")
 		}
 	}
-	aResp, err := suggester.AuthorsSuggester.GetSuggestions(payload, tid)
+	authorsResp, err := suggester.AuthorsSuggester.GetSuggestions(payload, tid)
```
Comment (andreic92) [inline service/suggester.go]: Should we make this call if `flags.AuthorsFlag == TmeSource`?

Comment (tosan88) [reply service/suggester.go]: Indeed, we could :+1:

Comment (andreic92) [reply service/suggester.go]: I think that I expressed myself wrong, I meant to say that performance wise, I guess that this should pe performed only when `flags.AuthorsFlag != TmeSource`. Correct me if I'm wrong.

File: service/suggester.go [hunk 2]
```diff
@@ -76,49 +82,55 @@ func NewAuthorsSuggester(authorsSuggestionApiBaseURL, authorsSuggestionEndpoint
...
-		for _, value := range fResp.Suggestions {
-			if isNotAuthor(value) {
-				//retain suggestion from falcon response
-				fResp.Suggestions[i] = value
-				i++
-			}
-		}
-		fResp.Suggestions = fResp.Suggestions[:i]
+
+	if flags.AuthorsFlag != TmeSource {
```
Comment (khatton-ft) [inline service/suggester.go]: This bit could be simplified with a `switch flag.AuthorsFlag`.
- case TmeSource: resp = falconResp
- case UppSource: remove TME authors, append UPP authors

File: service/suggester.go (hunk 1 above)
Comment (khatton-ft) [reply service/suggester.go]: I think we should only make this call if necessary.

**Pull request merged**